.github/workflows/validate_datasets.yml

```
@@ -60,13 +61,38 @@ jobs:
         bash -c "npm install -g bids-validator-*.tgz"
         popd
 
+    - uses: denoland/setup-deno@v1.1.2
```

Do we really want to pin so tightly?

```suggestion
    - uses: denoland/setup-deno@v1
```

---

I personally do not care ;-)